pkg/opentelemetry/output.go

```diff
@@ -36,47 +52,198 @@ func New(p output.Params) (*Output, error) {
 
 // Description returns a human-readable description of the output that will be shown in `k6 run`
 func (o *Output) Description() string {
-	return "OpenTelemetry: " + o.config.Address
+	return "opentelemetry: " + o.config.String()
 }
 
 // Stop flushes all remaining metrics and finalizes the test run
 func (o *Output) Stop() error {
```

Comment:
```suggestion
func (o *Output) StopWithTestError() error {
```
We should implement StopWithTestError instead of Stop(). Stop should just proxy the request.

Reply:
I'm not sure if I'm getting the idea or maybe missing the context; `Stop` is part of the interface. `StopWithTestError` is planned to replace `Stop` ?

Reply:
Yeah, that was the plan. https://github.com/grafana/k6/blob/153e0e505807da9e330fda816ce6d63a4bbf4691/output/types.go#L84-L85

Reply:
Good catch then, I'll adjust accordingly :+1: